### PR TITLE
uncapitalizing gonomics

### PR DESCRIPTION
**Pull request by cfauci**
# Description

[Description]

## Relevant Links

- [Display Text](https://www.vertgenlab.org/)

### Checklist before requesting a review

- [ ] I performed a self-review of my code
- [ ] If it this a core feature, I have added thorough tests
- [ ] The command `go fmt` was used on all files included

### Testing

None

### Screenshots & Media

None

### Edge cases / Breaking Changes / Known Issues

None

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -88,8 +88,8 @@ docker run -v $(pwd):/mnt gonomics go run $script.go
 
 ---
 
-### Compatibility with Previous Golang Versions
-<p>Gonomics is compatible with Golang version 1.18 and above. Please note that due to changes in random number generation since Golang v1.20,
+### Compatibility with previous golang Vvrsions
```
Comment (rimangan) [inline README.md:91]: small misprint 'vvrsions'

**Review by rimangan — APPROVED** [review]

**Review by hcnapier — APPROVED** [review]
